Seed.Api.IntegrationTests/Controllers/UserIntegrationTest.cs

```diff
@@ -0,0 +1,285 @@
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Seed.Api.IntegrationTests.Controllers.TestData;
+using Seed.Api.IntegrationTests.Generics;
+using Seed.Api.Models;
+using Seed.Data.EF;
+using Seed.Data.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Seed.Api.IntegrationTests.Controllers
+{
+    public class UserIntegrationTest : IClassFixture<ApiTestFixture>
```

Comment:
Minor but I would love to see the exact same naming convention for the test classes/filenames. For that reason this file (and class) should be `UserControllerTests`.